Automattic/sensei
# Enable to customize  sidebar-width and  header-height via css variables

Solves part of #[Learning Mode - Style tweaks for all Learning Mode templates#46](https://github.com/Automattic/sensei-theme/issues/46)


### Context
As part of the new theme initiative, we need to increase the learning mode ability to receive customizations.
This PR is ensuring that the left sidebar and the header height can be customized by CSS variables.

### Changes proposed in this Pull Request
* Enable to customize the Learning mode sidebar-width and header height variables via themes.

### Testing instructions
- Install a theme
- Create a course
- Enable the learning mode
- set new values sidebar-width and  for header-height`  on the  theme css (check if the theme css is loaded during the learning mode)  `--header-height` and  `--sidebar-width` 
- Check if the learning mode is using the new values.
